README.md

```diff
@@ -1,5 +1,3 @@
-
-
```

Comment:
I removed these empty lines because I had a bit of a scare when a build for a commit I was expecting to successfully report analytics did not do so.

That build just so happened to be a rebuild. The first build for that commit had analytics, the second did not. My assumption was that rebuilds did not report analytics, maybe because of a Test Analytics bug. I pushed a new commit to verify that.

I was happy to see that the build for this commit reported to Test Analytics. 

But... I was also quite confused when I looked at the builds for this branch and saw that the one that hadn't reported and sent me down this verification rabbit hole now had reports linked to it. 🤔 ... maybe it was just a reporting hiccup / delay then.

Anyways, I think this change is worth keeping, albeit out of scope.